src/peergos/shared/user/fs/MimeTypes.java

```diff
@@ -121,6 +123,10 @@ public static final String calculateMimeType(byte[] start, String filename) {
                 return "text/calendar";
             return "text/plain";
         }
+
+        if (filename.endsWith(UserContext.App.Todo.TODO_FILE_EXTENSION))
```

Comment:
I'd like to figure out a way to do this without relying on the extension, by making the cbor have a fixed header.
Probably having cbor = list(mimetype, map=data) will do it.

Reply:
done - but not sure about this